hydromt_sfincs/quadtree.py

```diff
@@ -1580,6 +1603,76 @@ def _find_boundary_cells(self):
 
         return bounds
 
+    def _find_boundary_cells_msk2(self):
```

Comment:
Think it would be good to review this part as well, and discuss with Maarten

Reply:
After internal discussion, this might not be needed after all. First check routines that determine the boundary cells in SnapWave.

Reply:
Agreed, but in order to proceed I would keep it in for now, since it doesn't do any harm, and remove later when checked in kernel